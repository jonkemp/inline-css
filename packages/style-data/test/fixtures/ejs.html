<html><link rel="stylesheet" href="ejs.css" />
<body>
<%= config.sitePrefix %>verify?grant=<%= grant._id %>&secret=<%= grant.secret %>
<div <%= config.sitePrefix %>></div></body></html>
