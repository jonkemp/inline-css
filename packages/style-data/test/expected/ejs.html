<html>
    <head><link rel="stylesheet" href="ejs.css"></head>
<body>
<%= config.sitePrefix %>verify?grant=<%= grant._id %>&secret=<%= grant.secret %>
<div <%= config.sitePrefix %>></div>
</body>
</html>
